.github/TRIAGERS.md

```diff
@@ -1,2 +1 @@
 # This file documents Triage members in the Llama Stack community
- @franciscojavierarceo
```

Comment:
should we perhaps remove the triager role if we are no longer going to have anyone actively in it?